docs/hosting/configuration.md

```diff
@@ -211,43 +205,38 @@ You can find the name of your timezone [here](https://momentjs.com/timezone/).
 
 ### User folder
 
-User-specific data like the encryption key, SQLite database file, and
-the ID of the tunnel (if used) gets saved by default in the subfolder
-`.n8n` of the user who started n8n. It is possible to overwrite the
-user-folder via an environment variable.
+n8n saves user-specific data like the encryption key, SQLite database file, and
+the ID of the tunnel (if used) in the subfolder `.n8n` of the user who started n8n. It's possible to overwrite the user-folder using an environment variable.
 
 ```bash
 export N8N_USER_FOLDER=/home/jim/n8n
 ```
 
 ### Webhook URL
 
-The webhook URL will normally be created automatically by combining
-`N8N_PROTOCOL`, `N8N_HOST` and `N8N_PORT`. However, if n8n runs behind a
-reverse proxy that would not work. That's because n8n runs internally
-on port 5678 but is exposed to the web via the reverse proxy on port 443. In
-that case, it is important to set the webhook URL manually so that it can be
-displayed correctly in the Editor UI and even more important is that the correct
-webhook URLs get registred with the external services.
+n8n creates the webhook URL by combining `N8N_PROTOCOL`, `N8N_HOST` and `N8N_PORT`. If n8n runs behind a reverse proxy that won't work. That's because n8n runs internally
```

Comment:
```suggestion
n8n creates the webhook URL by combining `N8N_PROTOCOL`, `N8N_HOST` and `N8N_PORT`. If n8n runs behind a reverse proxy, that won't work. That's because n8n runs internally
```
It lures me down the garden path without the comma 😅